src/sql/azdata.d.ts

```diff
@@ -5051,20 +5051,11 @@ declare module 'azdata' {
 		}
 
 		export interface INotebookMetadata {
-			kernelspec?: IKernelInfo | IKernelSpec | undefined;
+			kernelspec?: IKernelSpec | undefined;
```

Comment:
Normally we try to not change the finalized public APIs here - is removing IKernelInfo actually necessary? 

And while you're modifying this if you could add a doc comment that'd be great...

Reply:
That IKernelInfo interface has been marked as deprecated for ages, and was adding needless complexity when trying to access kernelSpec info.